todo_app/app.py

```diff
@@ -10,36 +11,36 @@
 @app.route('/items', methods=["GET"])
 @app.route('/', methods=["GET"])
 def index():
-    items = get_items()
+    items = fetch_todo_items()
     items = sorted(items, key=lambda x:(x.get("status")!='Not Started', items))
     return render_template('index.html', items = items)
 
 @app.route('/items/<id>', methods=["GET", "POST"])
 def get(id):
-    item = get_item(id)
+    item = fetch_todo_items(id)
     return render_template('saveItem.html', item = item)
 
 @app.route('/items/<id>/edit', methods=["GET", "POST"])
 def edit(id):
-    item = get_item(id)
+    item = get_to_do_list_id(id)
     if request.method=="POST":
         item["title"]=request.form.get('itemTitle')
         item["status"]=request.form.get('itemStatus')
         save_item(item)
         return redirect(url_for('get', id = item["id"]))
     return render_template('edit.html', item = item)
 
-
 @app.route('/items/new', methods=["POST"])
 def add():
     title = request.form.get('itemTitle')
-    add_item(title)
+    create_new_card(title)
```

Comment:
This function errors because it expects a description but you don't pass one through here. The best thing is probably to either implement the description or (more easily) remove it from the function below, but if we want to protect it for future usage then we might just use an empty argument for now (this is a bad habit to get into though!)
```suggestion
    create_new_card(title, None)
```